ui/cypress/e2e/single-search-page.cy.ts

```diff
@@ -150,6 +153,7 @@ describe("Tilavaraus ui search page (single)", () => {
     inputMaxPersons().should("not.be.visible");
     inputReservationUnitType().should("not.be.visible");
 
-    cy.a11yCheck();
+    // FIXME this was mis typed and it's not passing
+    // cy.checkA11y();
```

Comment:
It's a custom method created in `ui/cypress/support/commands.js`

Reply:
Ah good, the naming made me assume it's a typo.